### ISSUE TITLE
variable length wildcard behaves like `*`

### ISSUE THREAD

**Issue by Lewwwk** [superscriptjs/superscript#237]
Entering

> foo

with the following script

```
+ *(3-99)
- {keep}Caught by variable length

+ foo
- {keep}Direct match

+ *
- {keep}Caught by catchall
```

always returns

> Caught by variable length

**Comment by silentrob** [comment]
Okay, I have confirmed this is an issue.

**Comment by benhjames** [comment]
Should be fixed in v1.0.0. 😄
